.env.example

```diff
@@ -1,5 +0,0 @@
-PORT=4000
```

Comment:
I don't think you want to delete this file - it should be kept in the project so other developers can copy it when setting up their local env. Let me know if you need a hand restoring it.

Reply:
I have a `.env` with the same code I think?